lnbits/db.py

```diff
@@ -61,6 +63,8 @@ def _parse_timestamp(value, _):
 else:
     if os.path.isdir(settings.lnbits_data_folder):
         DB_TYPE = SQLITE
+        sqlite3.register_converter("json", lambda val: json.loads(val.decode("utf-8")))
+        sqlite3.register_converter("timestamp", lambda val: int(val))
```

Comment:
no, they are native in sqlite (timestamp already used before). 
The problem was that psycopg2 automatically decodes json cols into python objects by default but sqlite3 doesnt. This converter fixes that.
The timestamp converter is there because when `PRASE_DECLTYPES` (see below) is active the default timestamp converter sqlite3 has registered causes trouble. The `timestamp` converter makes sure the behaviour doesnt change.